tests/bug-6278-2/test.yaml

```diff
@@ -0,0 +1,17 @@
+requires:
+  min-version: 6
+
+pcap: false
+exit-code: 1
+args:
+  - --engine-analysis
+
+checks:
+  - shell:
+      args: grep -c 'user name cannot be set to an empty value' stderr
+      expect: 1
+      version: 7
```

Comment:
`min-version`.  When we move to 8, we still want this test to run.

Reply:
good point. Fixed